examples/tutorials/advanced/Federated Recurrent Neural Network.ipynb

```diff
@@ -0,0 +1,1133 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['Adamou', 'Agelakos', 'Akrivopoulos', 'Alexandropoulos', 'Anetakis', 'Angelopoulos', 'Antimisiaris', 'Antipas', 'Antonakos', 'Antoniadis', 'Antonopoulos', 'Antonopoulos', 'Antonopoulos', 'Arvanitoyannis', 'Avgerinos', 'Banos', 'Batsakis', 'Bekyros', 'Belesis']\n",
+      "['Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek', 'Greek']\n",
+      "\n",
+      " \n",
+      " Amount of data points loaded: 20074\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Set of names(X)\n",
+    "names_list = []\n",
+    "#Set of labels (Y)\n",
+    "category_list = []\n",
+    "\n",
+    "#Convert into a list with corresponding label.\n",
+    "\n",
+    "for nation, names in category_lines.items():\n",
+    "    #iterate over every single name\n",
+    "    for name in names:\n",
+    "        names_list.append(name)      #input data point\n",
+    "        category_list.append(nation) #label\n",
+    "        \n",
+    "#let's see if it was successfully loaded. Each data sample(X) should have its own corresponding category(Y)\n",
+    "print(names_list[1:20])\n",
+    "print(category_list[1:20])\n",
+    "\n",
+    "print(\"\\n \\n Amount of data points loaded: \" + str(len(names_list)))\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now need to turn our categories into numbers, as PyTorch cannot really understand plain text\n",
+    "\n",
+    "For an example category: \"Greek\" ---> 0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['Armbruster', 'Armbruster', 'Artz', 'Aue', 'Auer', 'Augustin', 'Aust', 'Autenburg', 'Auttenberg', 'Baasch']\n",
+      "[3 3 3 3 3 3 3 3 3 3]\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Assign an integer to every category\n",
+    "categories_numerical = pd.factorize(category_list)[0]\n",
+    "#Let's wrap our categories with a tensor, so that it can be loaded by LanguageDataset\n",
+    "category_tensor = torch.tensor(np.array(categories_numerical), dtype=torch.long)\n",
+    "#Ready to be processed by torch.from_numpy in LanguageDataset\n",
+    "categories_numpy = np.array(category_tensor)\n",
+    "\n",
+    "#Let's see a few resulting categories\n",
+    "print(names_list[1200:1210])\n",
+    "print(categories_numpy[1200:1210])\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now need to turn every single **character** in each input line string into a vector, with a \"1\" marking the character present in that very character.\n",
+    "\n",
+    "#Example: character \"a\" = array([[1., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
```

Comment:
Remove # before example. In markdown this # -> title, which is probably unintended in this case.

Reply:
Fixed